R/test-data.R

```diff
@@ -154,72 +156,139 @@ process_test_data <- function() {
+    dplyr::mutate(tests_cumulative = as.numeric(tests_cumulative)) %>%
+    dplyr::mutate(new_tests = as.numeric(new_tests)) %>%
+    dplyr::mutate(tests_cumulative_corrected = NA_real_) %>%
+    dplyr::mutate(new_tests_corrected = NA_real_) %>%
+    dplyr::mutate(date = as.Date(date)) %>%
+    dplyr::mutate(source = "fetch") %>%
+    dplyr::relocate(
+      country, tests_cumulative, new_tests,
+      tests_cumulative_corrected, new_tests_corrected,
+      date, source
+    )
+
+  manual_list <- sprintf(
+    "https://raw.githubusercontent.com/dsbbfinddx/FINDCov19TrackerData/master/manual/processed/%s-processed-manually.csv", # nolint
+    window_update
+  )
+
+  manual_tests_daily <- tryCatch(
     {
-  processed_manual <- readr::read_csv(sprintf("https://raw.githubusercontent.com/dsbbfinddx/FINDCov19TrackerData/master/manual/processed/%s-processed-manually.csv", today), # nolint
-    col_types = cols(
-      country = col_character(),
-      tests_cumulative = col_double(),
-      new_tests = col_double(),
-      tests_cumulative_corrected = col_double(),
-      new_tests_corrected = col_double(),
-      date = col_date(format = ""),
-      source = col_character()
-    ),
-    quoted_na = FALSE
-  ) # nolint
-      calculate_tests_manual_file(processed_manual)
+      fl_gh <- gh::gh("GET /repos/:owner/:repo/git/trees/master?recursive=1",
+                      owner = "dsbbfinddx",
+                      repo = "FINDCov19TrackerData",
+                      branch = "selenium"
+      )
+
+      # takes the manual files uploaded between
+      # the time frame to update (first_date and today)
+      filelist_manual <- unlist(lapply(fl_gh$tree, "[", "path"), use.names = FALSE) %>%
+        stringr::str_subset(., "manual/processed/.*processed-manually.csv$") %>%
+        paste0("https://raw.githubusercontent.com/dsbbfinddx/FINDCov19TrackerData/master/", .)
+
+      manual_files <- manual_list[which(manual_list %in% filelist_manual)]
+
+      processed_manual <- rio::import_list(manual_files, rbind = TRUE) %>%
+        dplyr::select(-`_file`, -status, -url) %>%
+        dplyr::mutate(tests_cumulative = as.numeric(tests_cumulative)) %>%
+        dplyr::mutate(new_tests = as.numeric(new_tests)) %>%
+        dplyr::mutate(tests_cumulative_corrected = as.numeric(tests_cumulative_corrected)) %>%
+        dplyr::mutate(new_tests_corrected = (new_tests_corrected)) %>%
+        dplyr::mutate(date = as.Date(date)) %>%
+        dplyr::mutate(source = "manually") %>%
+        dplyr::relocate(
+          country, tests_cumulative, new_tests,
+          tests_cumulative_corrected, new_tests_corrected,
+          date, source
+        )
     },
     error = function(cond) {
-      cli::cli_alert_info("No file with manual test countries found for
-          today. Ignoring input.", wrap = TRUE)
+      cli::cli_alert_info("Any file founded in folder manual/processed for the
+              input days. Ignoring uploaded manual tests.", wrap = TRUE)
```

Comment:
How about "no file found [...] for days $days. Ignoring manual test files for these days."?